### ISSUE TITLE
Error: External Program Execution Failed

### ISSUE THREAD

**Issue by kanan2u** [terraform-google-modules/terraform-google-cloud-operations#38]
### TL;DR

While running the script from https:://registry.terraform.io/modules/terraform-google-modules/cloud-operations/google/latest/submodules/agent-policy, I am getting the below error:

 with module.agent_policy.module.gcloud-upsert.data.external.env_override[0],
│   on .terraform\modules\agent_policy.gcloud-upsert\main.tf line 73, in data "external" "env_override":
│   73:   program = ["${path.module}/scripts/check_env.sh"]
│
│ The data source received an unexpected error while attempting to execute the program.
│
│ Program: .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh
│ Error: fork/exec .terraform/modules/agent_policy.gcloud-upsert/scripts/check_env.sh: %1 is not a valid Win32
│ application.

Can anyone please suggest on this?

### Expected behavior

Ops agent should be installed in the GCP compute machine (VM) while terraform apply is being executed.

### Observed behavior

Error occurred in the Terraform inbuilt modules as mentioned in the above error details.

### Terraform Configuration

```hcl
module "agent_policy" {
  source     = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version    = "~> 0.1.0"

  project_id = "dexxx-tuxxxx-xxxxxxx"
  policy_id  = "ops-agents-example-policy1"
  agent_rules = [
    {
      type               = "ops-agent"
      version            = "current-major"
      package_state      = "installed"
      enable_autoupgrade = true
    },
  ]
  group_labels = [
  ]
  os_types = [
    {
      short_name = "debian"
      version    = "9"
    },
  ]
    zones        = ["us-west4-b"]
}
```

### Terraform Version

```sh
Terraform v1.1.7
on windows_amd64
+ provider registry.terraform.io/hashicorp/external v2.2.2
+ provider registry.terraform.io/hashicorp/google v4.14.0
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/hashicorp/random v3.1.2
```
